sklearn/isotonic.py

```diff
@@ -190,6 +190,12 @@ class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
     X_max_ : float
         Maximum value of input array `X_` for right bound.
 
+    interpolation_X_ : array-like of shape (n_interpolation_points,)
+        Interpolation point of `X`.
+
+    interpolation_y_ : array-like of shape (n_interpolation_points,)
+        Interpolation point of `y`.
```

Comment:
`n_interpolation_points` is not defined. We should make it explicit that it depends on the data. It is always smaller than the number of samples in the training set because of the de-duplication of sorted (x, y) pairs.

Reply:
If the variables are renamed, `n_interpolations_points` should be renamed to `n_thresholds`.